js/scorm/wrapper.js

```diff
@@ -301,14 +306,31 @@ define ([
 
             this.endTime = new Date();
 
+            var exit_property = "cmi.core.exit";
+            var exit_value = "";
+            var is_complete = this.scorm.data.completionStatus === "completed" || this.scorm.data.completionStatus === "passed";
```

Comment:
`isComplete` would be more in keeping with the general style used in Adapt

Reply:
Is snakeCase the standard? If so, I can change the other new variables.